templates/taxi/car_detail.html

```diff
@@ -10,6 +10,14 @@
     <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
       Update
     </a>
+    <form action="" method="post">
```

Comment:
This is a code quality issue. The form action is empty. It would be better to specify the URL to which the form data should be sent when the form is submitted.